neqo-client/src/main.rs

```diff
@@ -40,12 +40,20 @@ pub struct Args {
     #[structopt(short = "h", long, number_of_values = 2)]
     header: Vec<String>,
 
-    #[structopt(name = "max-table-size", short = "t", long, default_value = "128")]
-    max_table_size: u32,
+    #[structopt(name = "max-table-capacity", short = "t", long, default_value = "128")]
+    max_table_capacity: u32,
 
     #[structopt(name = "max-blocked-streams", short = "b", long, default_value = "128")]
     max_blocked_streams: u16,
 
+    #[structopt(
+        name = "max-concurent_push-streams",
```

Comment:
Is the mingling of "-" and "_" intentional or a typo?

concurrent has two "r"s.